lib/inv_ca_cg.cpp

```diff
@@ -760,7 +795,7 @@ namespace quda {
       param.secs += profile.Last(QUDA_PROFILE_COMPUTE);
 
       // store flops and reset counters
-      double gflops = (blas::flops + mat.flops() + matSloppy.flops())*1e-9;
+      double gflops = (blas::flops + mat.flops() + matSloppy.flops() + matPrecon.flops()) * 1e-9;
```

Comment:
Remind me, can `matPrecon` and `matSloppy` alias each other? If so, is this math wrong?

Reply:
no they cannot

Reply:
and even if they did, no error, since the act of fetching the flops also zeros the counter